src/common/components/Typography/Typography.tsx

```diff
@@ -24,22 +52,71 @@ const Typography = ({
   variant = TypographyVariant.body1,
   uppercase = false,
   className,
-  customFontWeight,
-  customColor,
+  customColor = 'text-gray-23',
 }: TTypographyProps): JSX.Element => (
```

Comment:
Yes, I know we can create classes to use and we can use follow your suggestion. But since we try to use the tailwind as much as we can without creating a new class so that's why I haven't created a new class. 

And I have planned to update all text in the wallet to use the Typography component after merged this PR.

Reply:
> And I have planned to update all text in the wallet to use the Typography component after merged this PR.
I'm sorry, this is pointless IMHO.
Our wiki says "It’s recommended to use Tailwind css for styling rather than CSS Modules or Styled components."
Recommended in general with only purpose to ease and speed up development, not to blindly follow and not to restrict.
I'm very concerned. Let's avoid needless work please.

Reply:
OK, I will make an update in the next PR.